_posts/2021-06-29-high-level-graphs.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: High Level Graphs update
+author: Genevieve Buckley
+theme: twitter
+---
+{% include JB/setup %}
+
+## Executive Summary
+
+There is a lot of work happening in Dask right now on high level graphs. We'd like to share a snapshot of current work in this area. This post is for people interested in technical details of behind the scenes work improving performance in Dask. You don't need to know anything about it in order to use Dask.
+
+## Contents
+
+* [Brief background](#brief-background)
+* [Blockwise layers progress](#blockwise-layers-progress)
+* [A high level graph for `map_overlap`](#a-high-level-graph-for-`map_overlap`)
```

Comment:
wait are backticks actually allowed in the links??

Reply:
Uh, no
:rofl: Can't believe a cared about this punctuation problem enough to rant about it earlier... but not enough to remember it was a thing when I wrote this draft :rofl:

Reply:
Fixed now